resources/views/frontend/category.blade.php

```diff
@@ -0,0 +1,24 @@
+@extends('layouts.frontend')
+@section('content')
+    <div id="web">
+        <div id="subheader">
+            <h1>Categories</h1>
+        </div>
+        <div id="content">
+            <ul id="catsinbox">
+                @if(count($categories)>0)
+                    @php $i=1; @endphp
+                    @foreach($categories as $value)
```

Comment:
```suggestion
                    @foreach($categories as $index => $value)
```
Ta có thể viết ntn thay vì dùng @php như trên nhé =)